README.md

```diff
@@ -93,19 +62,16 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
-<!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
+- **HTML**
+- **CSS**
 - **[key_feature_3]**
```

Comment:
- [ ] Please delete this line if you are not adding a feature here. Also kindly note that HTML and CSS are not features but technologies. Features are the things that you have implemented in your application/ what your application does.  eg Mobile navigation menu, about section, tagline ..